omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -514,6 +539,16 @@
 
                         </excludeIf>
 
+                        <span class="OtherAppointmentReasonEnableDisable1">
```

Comment:
Because its not enabling and disabling reason for next appointment but the other reason for next appointment.